ldap/servers/plugins/retrocl/retrocl_po.c

```diff
@@ -424,6 +430,16 @@ entry2reple(Slapi_Entry *e, Slapi_Entry *oe, int optype)
     }
     slapi_entry_add_values(e, retrocl_changetype, vals);
 
+    /* Does this entry contain any excluded attributes */
+    attrs = oe->e_attrs;
+    while(attrs) {
+        if (retrocl_attr_in_exclude_attrs(attrs->a_type, sizeof(attrs->a_type))) {
+            slapi_log_err(SLAPI_LOG_PLUGIN, RETROCL_PLUGIN_NAME, "entry2reple - excluding attr (%s).\n", attrs->a_type);
+            attrlist_delete(&attrs, attrs->a_type);
+        } else
+            attrs = attrs->a_next;
```

Comment:
Please always use "complete" curly brackets for if/else:

    } else {
        attrs = attrs->a_next;
    }